app/views/layouts/application.html.erb

```diff
@@ -7,10 +7,10 @@
     <%= csrf_meta_tags %>
   </head>
   <body>
-
-    <%= render "donations/new" %>
-
-    <%= render "layouts/navbar" %>
+      <br><% if signed_in? %></br>
```

Comment:
Let's ✂️ the `<br>` tags here. If they're there because of layout, we can handle it with CSS instead. 😀